sw/device/lib/dif/dif_keymgr.h

```diff
@@ -0,0 +1,648 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_LIB_DIF_DIF_KEYMGR_H_
+#define OPENTITAN_SW_DEVICE_LIB_DIF_DIF_KEYMGR_H_
+
+/**
+ * @file
+ * @brief <a href="/hw/ip/keymgr/doc/">Key Manager</a> Device Interface
+ * Functions
+ */
+
+#include <stdint.h>
+
+#include "sw/device/lib/base/mmio.h"
+#include "sw/device/lib/dif/dif_warn_unused_result.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif  // __cplusplus
+
+/**
+ * Size definitions.
+ */
+#define DIF_KEYMGR_SW_BINDING_WORDS 4
+#define DIF_KEYMGR_SALT_WORDS 4
+#define DIF_KEYMGR_NUM_SHARES 2
+#define DIF_KEYMGR_SHARE_WORDS 8
```

Comment:
Optional: to avoid what are technically unnecessary macros, I think it would be ok to write something like
```c
enum {
  kDifKeymgrSwBindingWords = 4,
};
```

Me and @lenary have thought about doing this to avoid unnecessary macro names I don't think we should block on it.